### DISCUSSION TITLE
Acustica plugins not working

### DISCUSSION THREAD

**Discussion by MarkCZA** [apohl79/audiogridder#1007] [Help]
Hi, 

Have an issue with all Acustica plugs, they wont load into AG anymore, was working fine up until a few days ago!!

Have attached some screen grabs.

All 3rd party plugs load without an issue, it just seems to be Acustica, however, N4 loads with no problems!

Any ideas what would be causing this?

![Screen Grab](https://user-images.githubusercontent.com/112706000/188151947-a9d441bd-80a6-4bd1-8fee-83560896b425.JPG)
![photo_2022-09-02_15-04-14](https://user-images.githubusercontent.com/112706000/188151956-61b0ceb5-e7eb-489a-b825-b9ddf001ab2d.jpg)

**Comment by apohl79** [comment]
When you see this error, do you have a screen connected to your windows server and turned on (or do you have a vnc session running)? I have seen this only when no display is connected to windows, as the screen capturing can't open the device in that case.

Btw, your plugin in the screenshot is v1.1 and this error popup was introduced in 1.2. So make sure you are running the same version of plugin and server.

**Comment by MarkCZA** [comment]
Hi, thanks for the response!
I have a separate monitor connected to my server pc.
Yep, tried both versions of AG, screen shot from 1.1 and a screen shot from 1.2, 
I'm using the same version of the plugin and server, not different ones.

**Comment by MarkCZA** [comment]
Just to be 100% clear, the problem is just with AA plugins, everything else works perfectly!
